src/Upgrade/Application/Dto/IntegratorResponseDto.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace Upgrade\Application\Dto;
+
+class IntegratorResponseDto
+{
+    /**
+     * @var string
+     */
+    protected const MESSAGES_KEY = 'message-list';
+
+    /**
+     * @var string
+     */
+    protected const WARNINGS_KEY = 'warning-list';
+
+    /**
+     * @var array<mixed>
+     */
+    protected array $data = [];
+
+    /**
+     * @param array<mixed> $data
+     */
+    public function __construct(array $data = [])
+    {
+        $this->data = $data;
+    }
+
+    /**
+     * @return array<string>|null
+     */
+    public function getMessages(): ?array
```

Comment:
`get*` method name, usually mean return smth or fail.
for logic, that is used here, `findMessages()` more suitable.

The same for the `getWarnings()`

Reply:
For the purpose, we use the naming `getOrFail`